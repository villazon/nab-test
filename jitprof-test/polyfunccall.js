function foo(a, b){
  return a+b;
}

foo(1,2);
foo(2,3);
foo("a", "b");
foo("a", 3);

