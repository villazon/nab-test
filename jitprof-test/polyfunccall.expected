---------------------------
Report of Polymorphic Binary Operations:
* Polymorphic binary operation at (jitprof-test/polyfunccall.js:2:10:2:13)
	Hit: 1	Miss: 2
	Count: 2	types: float + float
	Count: 1	types: string + float
	Count: 1	types: string + string
Report of Polymorphic Unary Operations:
[****]PolyFun: 1
